vernac/comAssumption.ml

```diff
@@ -220,8 +220,17 @@ let do_assumptions ~program_mode ~poly ~scope ~kind ?user_warns ~inline l =
   let univs = Evd.check_univ_decl ~poly sigma udecl in
   declare_context ~try_global_assum_as_instance:false ~scope ~univs ?user_warns ~inline ctx
 
+let warn_context_outside_section =
+  CWarnings.create ~name:"context-outside-section"
+    ~category:CWarnings.CoreCategories.vernacular
+    ~default:CWarnings.AsError
+    Pp.(fun () -> strbrk "Use of \"Context\" outside sections behaves \
+                          as \"#[local] Parameter\" or \"#[local] \
```

Comment:
Isn't this false? Context auto declares typeclass instances but Axiom doesn't. ie
~~~coq
Class C := {}.

Axiom x : C.

Fail Type _ : C. (* could not find an instance for C *)

Context (y : C).

Type _ : C. (* y : C *)
~~~

Reply:
Indeed, thanks, fixed.